engine/when.py

```diff
@@ -23,8 +21,8 @@ def _add_parameter_strings(parameter, kind_to_argstrings, kind_to_callstrings):
         arg_string = f"**{parameter.name}"
         call_string = f"**{parameter.name}"
     if parameter.default is not inspect.Parameter.empty:
-        arg_string = f"{parameter_string}={parameter.default}"
-        call_string = f"{parameter_string}={parameter_string}"
+        arg_string = f"parameter_string={parameter.default}"
```

Comment:
Oh, I see. This has been fucked up 'n otiose for a while. The correct fix here is

```
        arg_string = f"{parameter.name}={parameter.default}"
        call_string = f"{parameter.name}={parameter.name}"
```